src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -36,11 +38,13 @@ class JsonAdaptedPerson {
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
+            @JsonProperty ("github username") String gitHubUsername,
```

Comment:
Maybe github_username would be more suitable for the json key
```suggestion
            @JsonProperty ("github-username") String gitHubUsername,
```

Reply:
Also including all the other occurences of `"github username"`.